test/dlc_tests/conftest.py

```diff
@@ -618,6 +606,42 @@ def delete_s3_artifact_copy():
     return conn
 
 
+@pytest.fixture(autouse=True)
+def skip_s3plugin_test(request):
+    if "training" in request.fixturenames:
+        img_uri = request.getfixturevalue("training")
+    elif "pytorch_training" in request.fixturenames:
+        img_uri = request.getfixturevalue("pytorch_training")
+    else:
+        return
+    _, fw_ver = get_framework_and_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_s3plugin_test"):
+        if Version(fw_ver) not in SpecifierSet("<=1.12.1,>=1.6.0"):
+            pytest.skip(
+                "s3 plugin is only supported in PT 1.6.0 - 1.12.1, skipping this container with tag{}".format(
+                    fw_ver
+                )
```

Comment:
Why not just use f-strings, like:
```suggestion
                f"s3 plugin is only supported in PT 1.6.0 - 1.12.1, skipping this container with tag{fw_ver}"
```

It would make the code more compact and make the formatting easier to read.